libs/features/personalization/personalization.js

```diff
@@ -1066,6 +1066,12 @@ export async function init(enablements = {}) {
     });
   }
 
+  if (target === true) {
+    const { preloadBlockResources } = await import('../../utils/utils.js');
+    const lcpBlocks = [...document.querySelectorAll('body > main > div:first-child > *[class]')];
+    preloadBlockResources(lcpBlocks, false);
```

Comment:
Can you really know which blocks are relevant before target loads? What if the markup is changed and the blocks you are loading are throw-away work, because target changes the site?